src/testsuite/vec_bcd_dummy.c

```diff
@@ -4,7 +4,7 @@
  *  Created on: Mar 1, 2018
  *      Author: sjmunroe
  */
-
+#ifndef __clang__
```

Comment:
add a comment here?

Reply:
OK
// Clang support for BCD is disabled until clang supports _Decimal128.